src/Web/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -43,5 +43,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Context>BrowseObject</Context>
     </PropertyPageSchema>
   </ItemGroup>
-
-</Project>
\ No newline at end of file
+  
+  <!-- Web specific target to ensure that generated content (css, js, etc.) is copied to output -->
+  <Target Name="_IncludeWebProjectGeneratedContent" BeforeTargets="GetCopyToPublishDirectoryItems" Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultContentItems)' == 'true' ">
```

Comment:
Yes please. And can we rename the target name to something like  _IncludePrePublishGeneratedContent (or something on that lines) to denote that this target will include the contents generated before the actual publish.